pkg/kudoctl/cmd/init_test.go

```diff
@@ -27,7 +27,7 @@ import (
 	"github.com/kudobuilder/kudo/pkg/kudoctl/util/repo"
 )
 
-var updateGolden = flag.Bool("update", false, "update .golden files and manifests in /config/crd")
+var updateGolden = flag.Bool("update", true, "update .golden files and manifests in /config/crd")
```

Comment:
Please set this back to false. Otherwise we might have tests succeeding that should fail. Ideally, updating golden files should be separate from any test setup.